src/js/pensions/config/form.js

```diff
@@ -300,10 +300,10 @@ const formConfig = {
                 'ui:options': {
                   labels: {
                     Longevity: 'Longevity',
-                    PDRL: 'PDRL',
                     Separation: 'Separation',
```

Comment:
We can probably take these entries out. I don't think you have to specify labels in here for everything.